tools/plugins/virtual-node/src/main/kotlin/net/corda/cli/plugins/vnode/commands/ResetCommand.kt

```diff
@@ -89,4 +103,20 @@ class ResetCommand : HttpRpcCommand(), Runnable {
             println("CPI Successfully Uploaded and applied. ")
         }
     }
+
+    private fun resyncVaults(virtualNodeShortIds: List<String>) {
+        createHttpRpcClient(VirtualNodeMaintenanceRPCOps::class).use { client ->
+            val virtualNodeMaintenance = client.start().proxy
+            try {
+                virtualNodeShortIds.forEach { virtualNodeShortId ->
+                    virtualNodeMaintenance.resyncVirtualNodeDb(virtualNodeShortId)
```

Comment:
Should we:
```suggestion
                    virtualNodeMaintenance.resyncVirtualNodeDb(virtualNodeShortId.uppercase())
```
to make sure that through the all sub-systems it is travelling in the expected case?

Probably even better if this is done on the HTTP RPC implementation side to make sure it works even if it invoked through Swagger UI.

Reply:
I was worried about doing it at the point of ingress, but if you think that's a better thing to do then I'm happy to switch over